patroni/api.py

```diff
@@ -556,7 +556,12 @@ def __httpserver_init(self, host, port):
         info.sort(key=lambda x: x[0] == socket.AF_INET, reverse=not dual_stack)
 
         self.address_family = info[0][0]
-        HTTPServer.__init__(self, info[0][-1][:2], RestApiHandler)
+        try:
+            HTTPServer.__init__(self, info[0][-1][:2], RestApiHandler)
+        except OSError:
```

Comment:
I think it should be the `socket.error`, otherwise python2 will miss it.

Reply:
Yes, you are right, fixed